kubernetes/platform-services/values.yaml

```diff
@@ -26,69 +26,86 @@ configServicePort: &configServicePort 9012
 gatewayServiceHost: &gatewayServiceHost hypertrace-data-query-service
 gatewayServicePort: &gatewayServicePort 9001
 
-hypertrace-oc-collector:
+hypertrace-collector:
   resources:
-    requests:
-      cpu: "0.5"
-      memory: "256Mi"
     limits:
-      cpu: "1"
-      memory: "256Mi"
-  containerPorts:
-    - name: grpc-opencensus
-      containerPort: 55678
-    - name: http-jaeger
-      containerPort: 14268
-    - name: jaeger-tchannel
-      containerPort: 14267
-    - name: zipkin
-      containerPort: 9411
+      cpu: 1
+      memory: 2Gi
```

Comment:
2Gi, does this collector has higher mem requirements compared to previous?

Reply:
These are the default values we are using. I am not sure if @pavolloffay and @jcchavezs have tried tuning it even further. But yeah, we can eventually tune it as per our experience.

Reply:
No my point is around increasing that default value. For oc collector the memory limit was 256Mi, is there a reason behind increasing it to 2Gi?

Reply:
By default values I mean, default helm values for hypertrace collector which you can find here: https://github.com/hypertrace/hypertrace-collector/blob/main/helm/values.yaml

Reply:
Ah, I see